test/e2e/pkg/testnet.go

```diff
@@ -663,3 +709,50 @@ func (g *ipGenerator) Next() net.IP {
 	}
 	return ip
 }
+
+func loadZoneLatenciesMatrix(filePath string) (map[ZoneID][]uint32, error) {
+	records, err := readCsvFile(filePath)
```

Comment:
Since we need to parse the csv file here anyway? Wouldn't it be easier to set the latencies from here? (rather than from a script inside the container) It would be easier to debug.
This is just a suggestion, not sure it's worth porting the script from python to go.